### DISCUSSION TITLE
可以添加JPEG格式识别吗

### DISCUSSION THREAD

**Discussion by Darthtoroyah** [lanyeeee/bilibili-manga-watermark-remover#17] [General]
谢谢

**Comment by lanyeeee** [comment]
你是指漫画原图的格式是`.jpeg`，然后选择漫画目录后，图片没被识别到，对吗

**Reply by Darthtoroyah** [reply]
是的

**Comment by lanyeeee** [comment]
已解决 #19 
相关功能将在下个版本发布，急用请自行编译develop分支

**Comment by lanyeeee** [comment]
相关功能已在0.4版本发布
https://github.com/lanyeeee/bilibili-manga-watermark-remover/releases/tag/0.4
